portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -11,8 +11,8 @@
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+  public void doGet(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
```

Comment:
Can you please explain what adding final does?

Reply:
It means that inside the method the variables can not be reassigned
I guess for avoid errors in the request

Reply:
I guess I'm confused because my understanding is that "final" means that the variable can't be modified, but you are modifying response by setting its content type? And it seems like the examples don't include the final keyword: https://github.com/JesusU076/software-product-sprint/blob/main/walkthroughs/week-2-server/examples/page-view-counter/src/main/java/com/google/sps/servlets/PageViewServlet.java#L30 Not saying you're wrong, just want to understand where these changes came from!

Reply:
No, you're right final means that the variable can't be modified, but when is a keyword for a method the it's means that the variable can't change i the method.

And now that I think about it now I am seriously wondering where I got the "final"  because as far as I know I got it from that example but apparently not. But it's not a bad feature

Reply:
Got it, ok--thanks for the explanation. I also just read https://stackoverflow.com/questions/2236599/final-keyword-in-method-parameters, so I don't think it's necessary here, but if you like it, then that's fine! Will approve + merge. If you ever change your mind, you can just address it in a future PR but can also leave as is if it's not causing any issues.